Project.toml

```
@@ -33,6 +33,7 @@ NNlib = "0.8.2"
 NNlibCUDA = "0.2"
 ProgressLogging = "0.1"
 Reexport = "0.2, 1.0"
+SpecialFunctions = "2"
```

Dep added in #1877, without a compat bound. 

Probably we should allow 1 too, these are the latest tags:
```suggestion
SpecialFunctions = "1.8.2, 2.1.2"
```